server/src/main/java/org/apache/iotdb/db/qp/strategy/LogicalGenerator.java

```diff
@@ -1169,9 +1170,15 @@ private PartialPath parseFullPath(FullPathContext ctx) {
   public void enterAttributeClauses(AttributeClausesContext ctx) {
     super.enterAttributeClauses(ctx);
     String dataType = ctx.dataType().getChild(0).getText().toUpperCase();
-    String encoding = ctx.encoding().getChild(0).getText().toUpperCase();
     createTimeSeriesOperator.setDataType(TSDataType.valueOf(dataType));
-    createTimeSeriesOperator.setEncoding(TSEncoding.valueOf(encoding));
+
+    TSEncoding encoding = TSEncoding.PLAIN;
```

Comment:
We could use the default Encoding method for each data type. 

![image](https://user-images.githubusercontent.com/7240743/95290145-221bc680-089f-11eb-84ea-e3f1a700707e.png)

Reply:
OK, I will assign a default value according to the data type